secrethub/resource_access_rule.go

```diff
@@ -33,10 +33,35 @@ func resourceAccessRule() *schema.Resource {
 				Description: "The permission that the account has on the given directory: read, write or admin",
 			},
 		},
+		Importer: &schema.ResourceImporter{
+			State: resourceAccessRuleImport,
+		},
+	}
+}
+
+func resourceAccessRuleCreate(d *schema.ResourceData, m interface{}) error {
+	provider := m.(providerMeta)
+	client := *provider.client
+
+	path := d.Get("dir_path").(string)
+	permission := d.Get("permission").(string)
+	account := d.Get("account_name").(string)
+
+	_, err := client.AccessRules().Get(path, account)
+	if err == nil {
+		return fmt.Errorf("access rule already exists: %s:%s", path, account)
```

Comment:
Would be nice to mention the import command: `access rule already exists: %s:%s, use the import command to proceed`

Reply:
What happens if you have the access rule defined twice in Terraform, e.g.

```terraform
resource "secrethub_access_rule" "one" {
  dir_path     = "path/to/dir"
  account_name = "alice"
  permission   = "read"
}

resource "secrethub_access_rule" "two" {
  dir_path     = "path/to/dir"
  account_name = "alice"
  permission   = "write"
}
```

In that case, you wouldn't want to import.

Reply:
Okay, let's leave it out.